.github/workflows/black.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
No sure exactly what you are doing in points 2) and 3) in description, but do 1 (copyright headers) in a specific PR please

Reply:
Alright limited this PR to (1). Thanks.